services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/client_api.py

```diff
@@ -151,6 +155,77 @@ async def begin_service_destruction(self, dynamic_sidecar_endpoint: str) -> None
             log_httpx_http_error(url, "POST", traceback.format_exc())
             raise e
 
+    async def service_state_save(self, dynamic_sidecar_endpoint: str) -> None:
+        url = get_url(dynamic_sidecar_endpoint, "/v1/containers:save-state")
+        try:
+            async with httpx.AsyncClient(timeout=self._save_restore_timeout) as client:
+                response = await client.put(url)
```

Comment:
PUT ? are you sure? should that not be a POST?

Reply:
changed as discussed below with PC